### ISSUE TITLE
no match found on every song

### ISSUE THREAD

**Issue by achinverma** [deezer/GraceNote2Deezer#1]
hi i am getting 
no match found on every song
can you tell me what is wrong, after downloading demo i have inserted GRACENOTE_APP_ID = " ";
in GraceNoteHelper.java
is there anything else i have to change please let me know?

**Comment by xgouchet** [comment]
You need to have a valid GraceNote App ID to make this sample work. To do that, check out the GraceNote Developer Portal : https://developer.gracenote.com/

**Comment by achinverma** [comment]
yes i have created account and app on  https://developer.gracenote.com/ and i have checked that my GRACENOTE_APP_ID is fine, but still i am getting no match found on every song ?
can you send me your email , i will send you my demo zip file

**Comment by jonx** [comment]
I suggest you try to use the default sample included in the Gracenote SDK. Then if it works, just diff with this project to add the Deezer integration.
